src/globus_cli/commands/timer/resume.py

```diff
@@ -43,21 +43,14 @@ def resume_command(
     Resume a timer.
     """
     timer_client = login_manager.get_timer_client()
-    job_doc = timer_client.get_job(timer_id)
+    timer_doc = timer_client.get_job(timer_id)
 
-    gare = _get_inactive_reason(job_doc)
+    gare = _get_inactive_reason(timer_doc)
     if not skip_inactive_reason_check:
         check_inactive_reason(login_manager, timer_id, gare)
 
-    resumed = timer_client.resume_job(
-        timer_id,
-        update_credentials=(gare is not None),
-    )
-    display(
-        resumed,
-        text_mode=TextMode.text_raw,
-        simple_text=resumed["message"],
-    )
+    resumed = timer_client.resume_job(timer_id, update_credentials=(gare is not None))
+    display(resumed, text_mode=TextMode.text_raw, simple_text=resumed["message"])
```

Comment:
I would (actually _quite strongly_) prefer that we not remove trailing commas in cases like this, which causes `black` to format onto a single line.
Leaving trailing commas -- and in many cases I intentionally add them for this -- makes `black` split arguments over multiple lines. The result is one of the many benefits of `black`: minimal diffs.

If a keyword argument is added here, I should only have to review that addition, not the potential refolding of text caused by extending past the 88 char limit.

Let's please revert this part of the change and any other updates like this which aren't part of renaming.

Reply:
To be fair, I didn't simply remove the trailing comma -- I manually reformatted it myself because it fit on a single line. I didn't realize you added trailing commas specifically to force things onto multiple lines, and I'll try to respect that.

I'll undo this change.

Reply:
This should be undone (simply kicked the change out of the previous commit and force-pushed).